src/interfaces.ts

```diff
@@ -0,0 +1,55 @@
+export interface LEVLEICredential {
```

Comment:
Why the difference in naming for Credentials vs Edges?

Reply:
I am not sure I understand the question. 
The data in the vlei is different than the data in the edges.

Reply:
`LEVLEICredential` or `LE_VLEI_CREDENTIAL`
`LE_VLEI_EDGES` or `LEVLEIEdges`

I think:

`LEvLEICredential` and
`LEvLEIEdges` would be better. 

Same for the ECR/OOR naming.